client/packages/host/src/components/Sync/Sync.tsx

```diff
@@ -0,0 +1,113 @@
+import React, { PropsWithChildren, useState, useEffect } from 'react';
+
+import {
+  Grid,
+  LoadingButton,
+  RadioIcon,
+  Typography,
+  useFormatDateTime,
+  useTranslation,
+} from '@openmsupply-client/common';
+import { useHost } from '../../api/hooks';
+import { SyncProgress } from '../SyncProgress';
+
+const STATUS_POLLING_INTERVAL = 1000;
+
+const useSync = () => {
+  const { localisedDate, localisedTime } = useFormatDateTime();
+  // Polling whenever Sync page is opened
+  const { data } = useHost.utils.syncInfo(STATUS_POLLING_INTERVAL);
+  const { mutateAsync: manualSync } = useHost.sync.manualSync();
+
+  // Derived state
+  const [latestSyncDate, setLatestSyncDate] = useState<null | string>(null);
```

Comment:
this shouldn't be state it's presentation.. just format it while rendering or at the very most, `const latestSyncDate = useMemo(() => ..., [data?.syncStatus]).`

Reply:
🙏 [like this](https://diffy.org/diff/e5e3821124448) ? 

Btw what I really wanted (keen to do later, not sure how to with translations), is a quick readable duration since last sync: `5 mins` or `10 seconds` or `2 hours 4 minutes` or `1 day 2 hour`

Reply:
like the `date-fns` `formatRelative` perhaps? that supports intl